lib/handlers/compile.js

```diff
@@ -42,14 +42,22 @@ function initialise(compilerEnv) {
     hasSetUpAutoClean = true;
     const tempDirCleanupSecs = compilerEnv.ceProps("tempDirCleanupSecs", 600);
     logger.info(`Cleaning temp dirs every ${tempDirCleanupSecs} secs`);
+
+    let cyclesBusy = 0;
```

Comment:
One day I should get the #1523 task done; this would be a prime candidate for stats.